contribute an esp32-wroom-32d case design (#368)
klaasnicolaas/home-assistant-glow: docs/docs/reference/cases.mdx

| ---- | ----------- | -- |
| ESP32S - Top | Top part of the case for the ESP32S | [Download](/cases/ESP32S/ESP32_top_side_v1.stl)
| ESP32S - Bottom | Bottom part of the case for the ESP32S | [Download](/cases/ESP32S/ESP32_bottom_side_v1.stl)

## ESP32-WROOM-32D

<p align="left">
  <img width="40%" src={require('@site/static/cases/ESP32-WROOM-32D/esp32-wroom-32d_inner_view.jpg').default} />
  <img width="40%" src={require('@site/static/cases/ESP32-WROOM-32D/esp32-wroom-32d_assembled_view.jpg').default} />
</p>

This case consists of 2 parts, a top and bottom where you can friction fit the pieces over each other and is designed to be used in both an indoor and outdoor environment. The LED status light is designed to slide into the housing and form a relatively tight seal. The USB cable is designed to slide up to provide some protection to water intrusion.

> If you are familiar with OnShape, you can find the model by searching for \"home-assistant-glow esp32-wroom-case\" where you can make a copy and modify it to your specifications.

Author: Yes, in created a share url and added it which should be durable and added it as an option.  I am not exactly sure how mdx files are used, so please check that I created a link successfully.

> If you are familiar with OnShape, you can find the model by searching for \"home-assistant-glow esp32-wroom-case\" where you can make a copy and modify it to your specifications, or by using this link [to the model.](https://cad.onshape.com/documents/28a914bca0ab451bdef13eb6/w/61ae11d2b0e90a508761a159/e/b81866b90bb3011f3cadb1f2).  An account may be required.

Reviewer: ```suggestion
If you're familiar with OnShape, you can access the model using [this link](https://cad.onshape.com/documents/28a914bca0ab451bdef13eb6/w/61ae11d2b0e90a508761a159/e/b81866b90bb3011f3cadb1f2). Simply make a copy and customize it according to your specifications. Please note that having an account might be necessary to make adjustments.
```

| ---- | ----------- | -- |
| ESP32S - Top | Top part of the case for the ESP32S | [Download](/cases/ESP32S/ESP32_top_side_v1.stl)
| ESP32S - Bottom | Bottom part of the case for the ESP32S | [Download](/cases/ESP32S/ESP32_bottom_side_v1.stl)

> 

Reviewer: ```suggestion
```

## ESP32-WROOM-32D

<p align="left">
  <img width="40%" src={require('@site/static/cases/ESP32-WROOM-32D/esp32-wroom-32d_inner_view.jpg').default} />
  <img width="40%" src={require('@site/static/cases/ESP32-WROOM-32D/esp32-wroom-32d_assembled_view.jpg').default} />
</p>

> This case consists of 2 parts, a top and bottom where you can friction fit the pieces over each other and is designed to be used in both an indoor and outdoor environment. The LED status light is designed to slide into the housing and form a relatively tight seal.  The USB cable is designed to slide up to provide some protection to water intrusion.

Reviewer: ```suggestion
This case consists of 2 parts, a top and bottom where you can friction fit the pieces over each other and is designed to be used in both an indoor and outdoor environment. The LED status light is designed to slide into the housing and form a relatively tight seal. The USB cable is designed to slide up to provide some protection to water intrusion.
```

If you are familiar with OnShape, you can find the model by searching for \"home-assistant-glow esp32-wroom-case\" where you can make a copy and modify it to your specifications.

| Name | Description | ⬇️ |
| ---- | ----------- | -- |

> | ESP32S - Top | Top part of the case for the ESP32D | [Download](/cases/ESP32-WROOM-32D/ESP32-WROOM-32D_top_case.stl)

Reviewer: ```suggestion
| ESP32 WROOM - Top | Top part of the case for the ESP32D | [Download](/cases/ESP32-WROOM-32D/ESP32-WROOM-32D_top_case.stl)
```

> | ESP32S - Bottom | Bottom part of the case for the ESP32D | [Download](/cases/ESP32-WROOM-32D/ESP32-WROOM-32D_bottom_case.stl)

Reviewer: ```suggestion
| ESP32 WROOM - Bottom | Bottom part of the case for the ESP32D | [Download](/cases/ESP32-WROOM-32D/ESP32-WROOM-32D_bottom_case.stl)
```
